lib/playground.dart

```diff
@@ -535,17 +535,16 @@ class Playground implements GistContainer, GistController {
     });
   }
 
-  /*
   /// Switch to the console or html results tab depending on whether the sample
   /// has html content or not.
-  void _switchOutputTab(String html) {
-    if (html.trim().isEmpty) {
+  void _switchOutputTab(String html, String dart) {
+    if (html.trim().isEmpty || !dart.contains('dart:html')) {
```

Comment:
I'd be a little more specific here, matching for the two strings `'dart:html'` and `"dart:html"` - surrounded by single and double quotes.

You may also want to re-do the if statement to make the logice more clear - matching on html switches to the html tab, matching on the imports statement switches to the html tab, otherwise switch to the console tab.